idd/Energy+.idd.in

```diff
@@ -36464,6 +36464,776 @@ ZoneHVAC:Baseboard:RadiantConvective:Electric,
        \minimum 0
        \maximum 1
 
+ZoneHVAC:CoolingPanel:RadiantConvective:Water,
+  A1 , \field Name
+       \required-field
+  A2 , \field Available Schedule Name
```

Comment:
@Myoldmopar Where appropriate with these changes to idd.in (this one and ones below), I am also updating the InputOutputReference section, if necessary.